tests/component-tests/textfieldControl/html/TextfieldStepper-test.html

```diff
@@ -62,5 +61,7 @@ <h2 id="qunit-userAgent"></h2>
         <button class="flc-textfieldStepper-increase">+</button>
     </div>
 
+    <script src="../js/TextfieldStepperTests.js"></script>
```

Comment:
I wonder if we could use [`defer`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer) for these cases.

Reply:
Good suggestion - this seems to work